src/MapfileStyleParser.ts

```diff
@@ -526,9 +563,10 @@ export class MapfileStyleParser implements StyleParser {
 
     if (typeof mapfileStyle.symbol === 'object') {
       const symbolType = mapfileStyle.symbol.type;
-      switch (symbolType) {
+      switch (symbolType.toLowerCase()) {
```

Comment:
the indentation seems off here

Reply:
if i format the switch cases get intended, which result in a lint error.